# Illumina Infinium HumanMethylation BeadChips Annotation üòÉ

## üßê About
The repository provides unambiguous probes (i.e., probes uniquely mapping to the target region) for Illumina 450K, EPIC/850K, and EPICv2.0/935K methylation arrays. These probes are evaluated using T2T-CHM13, the pangenome, and a combination of 95 assemblies (including T2T-CHM13 and 94 assemblies from the pangenome). Cross-population and population-specific unambiguous probes identified using the pangenome reference are also available. Importantly, we offer T2T-CHM13-based annotation information for each unambiguous probe, which can be leveraged to infer the potential biological functions of DNA methylation alterations in Epigenome-Wide Association Studies.

## üëØ Methylation arrays
<img align="right" alt="img" src="https://github.com/user-attachments/assets/47086cb5-d0e1-4373-9860-ff071b614526" width="50%" height="auto" />
The Illumina Infinium BeadChip array is the most widely used tool for characterizing genome-wide DNAm in humans, designed to measure DNAm level by the specific hybridization of each probe to the target DNAm site. The Illumina Infinium BeadChip array has a number of advantages in that it can be used
to assess patterns of DNAm with single-nucleotide resolution, is easy to use, is cost-effective,
and has high specificity and sensitivity. In addition, the arrays outlined above consistently
showed high reproducibility across technical and biological replicates, and demonstrated strong
correlations both with each other and with WGBS data. Given these advantages, DNAm arrays are widely used for genome-wide DNAm profiling in human
populations.

## :computer: Unambiguous probes
<b>T2T-based:</b>

    430,719 unambiguous probes on the HM450K array
    777,491 unambiguous probes on the EPIC (v1.0) array
    859,216 unambiguous probes on the EPIC v2.0 array

<b>Pangenome-based:</b>

    405,825 cross-population unambiguous probes on the HM450K array
    735,973 cross-population unambiguous probes on the EPIC (v1.0) array
    822,254 cross-population unambiguous probes on the EPIC v2.0 array

    405,825 AFR-specific unambiguous probes on the HM450K array
    735,973 AFR-specific unambiguous probes on the EPIC (v1.0) array
    822,254 AFR-specific unambiguous probes on the EPIC v2.0 array    

    405,825 AMR-specific unambiguous probes on the HM450K array
    735,973 AMR-specific unambiguous probes on the EPIC (v1.0) array
    822,254 AMR-specific unambiguous probes on the EPIC v2.0 array    

    405,825 EAS-specific unambiguous probes on the HM450K array
    735,973 EAS-specific unambiguous probes on the EPIC (v1.0) array
    822,254 EAS-specific unambiguous probes on the EPIC v2.0 array    

    405,825 EUR-specific unambiguous probes on the HM450K array
    735,973 EUR-specific unambiguous probes on the EPIC (v1.0) array
    822,254 EUR-specific unambiguous probes on the EPIC v2.0 array  

    405,825 SAS-specific unambiguous probes on the HM450K array
    735,973 SAS-specific unambiguous probes on the EPIC (v1.0) array
    822,254 SAS-specific unambiguous probes on the EPIC v2.0 array  



## ‚ö° Technologies
These unambiguous probes are developed using the concepts and methods described in Dong et al. (2024)

<p>
	<img width="50%" align="right" src="https://github-readme-stats.vercel.app/api?username=functionalepigenomics&show_icons=true&hide_border=true" />

## üì´ Contact Me
- Email: dzheng.th@gmail.com

